src/modules/UI/scenes/WalletList/WalletListConnector.js

```diff
@@ -25,6 +25,8 @@ const mapStateToProps = (state: State) => {
   const dimensions = state.ui.scenes.dimensions
   const customTokens = state.ui.settings.customTokens
   const otpResetPending = SETTINGS_SELECTORS.getOtpResetPending(state)
+  const progressPercentage = UI_SELECTORS.getWalletLoadingPercent(state)
```

Comment:
you call it percent in some places, percentage in others, ratio in others
Pick the most accurate term and use it consistently

Reply:
I will keep this in mind going forward